content/en/guides/training/serverless-rl/_index.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Dumb question: Do we plan to have more products with Training? If nay, consider removing nesting Serverless RL and instead moving the content up one level within `/training`

Reply:
@ngrayluna Yep, there are gonna be more products for Training. I already tried to nix the nesting despite that and people were not having it.